### PR TITLE
1.19.4 compatible

### PR DESCRIPTION
**Pull request by venraij**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,54 +1,58 @@
...
+    //
+    // You must be aware of the Mojang license when using the 'official' or 'parchment' mappings.
+    // See more information here: https://github.com/MinecraftForge/MCPConfig/blob/master/Mojang.md
+    //
+    // Parchment is an unofficial project maintained by ParchmentMC, separate from MinecraftForge
+    // Additional setup is needed to use their mappings: https://parchmentmc.org/docs/getting-started
+    //
+    // Use non-default mappings at your own risk. They may not always work.
+    // Simply re-run your setup task after changing the mappings to update your workspace.
+    mappings channel: 'official', version: '1.19.4'
```
Comment (mcenderdragon) [inline build.gradle]: please use parchment mappings, they have param Names

File: build.gradle [hunk 2]
```diff
@@ -57,14 +61,33 @@ minecraft {
...
+            property 'forge.logging.markers', 'REGISTRIES'
 
-            // Recommended logging level for the console
             property 'forge.logging.console.level', 'debug'
 
+            property 'forge.enabledGameTestNamespaces', 'examplemod'
+
             mods {
-                defaultdimension {
+                examplemod {
```
Comment (mcenderdragon) [inline build.gradle]: nope, remove the exampole mod and sue the proper mod ID

File: build.gradle [hunk 3]
```diff
@@ -73,32 +96,61 @@ minecraft {
...
-            // Recommended logging level for the console
             property 'forge.logging.console.level', 'debug'
 
-            args '--mod', 'examplemod', '--all', '--output', file('src/generated/resources/')
+            // Specify the modid for data generation, where to output the resulting resource, and where to look for existing resources.
+            args '--mod', 'examplemod', '--all', '--output', file('src/generated/resources/'), '--existing', file('src/main/resources/')
 
             mods {
-                defaultdimension {
+                examplemod {
```
Comment (mcenderdragon) [inline build.gradle]: right modID please.

File: build.gradle [hunk 4]
```diff
@@ -73,32 +96,61 @@ minecraft {
...
+    }
+}
+
 dependencies {
-    minecraft 'net.minecraftforge:forge:' + mc_version + '-' + forge_version
+    // Specify the version of Minecraft to use. If this is any group other than 'net.minecraft', it is assumed
+    // that the dep is a ForgeGradle 'patcher' dependency, and its patches will be applied.
+    // The userdev artifact is a special name and will get all sorts of transformations applied to it.
+    minecraft 'net.minecraftforge:forge:1.19.4-45.1.16'
+	implementation fg.deobf("curse.maven:jei-238222:4592504")
```
Comment (mcenderdragon) [inline build.gradle]: please sue the official JEI maven and not curseforge as maven... (beucase thuis way its harder to see what jei version is used) - and jei a´has a amven with javadocs and everything

**Review by mcenderdragon — CHANGES_REQUESTED** [review]
Huge thank you for porting. THis workls wuthout issues ? I had always the problem fo mayn different dead lcoks somehwere dead in minecraft code on new worls. and modded dimensions seemed not to always work.

File: build.gradle (hunk 1 above)
Comment (venraij) [reply build.gradle:29]: Worked fine for me even with modded dimensions

**Comment by venraij** [comment]
Worked fine for me even with modded dimensions, will fix the comments

**Comment by venraij** [comment]
@mcenderdragon changes requested have been implemented

**Review by mcenderdragon — APPROVED** [review]

**Comment by mcenderdragon** [comment]
I created a 1.19.x branch please target that oen isntead and then we are ready to ger this one merged! huge thank you!

**Comment by venraij** [comment]
@mcenderdragon  correct branch set as target

**Pull request merged**